File: platforms/swift/examples/scene/light/inner_cone_angle.swift
import FragmentColor

let torch = try Light.spot([0.0, 1.8, 1.0], [0.0, -1.0, 0.0], [1.0, 1.0, 1.0]).setConeAngles(0.15, 0.4)
let lamp = try Light.point([0.0, 0.0, 0.0], [1.0, 1.0, 1.0])

File: platforms/kotlin/examples/scene/light/inner_cone_angle.kt
import org.fragmentcolor.*

val torch = Light.spot(listOf(0.0f, 1.8f, 1.0f), listOf(0.0f, -1.0f, 0.0f), listOf(1.0f, 1.0f, 1.0f)).setConeAngles(0.15f, 0.4f)
val lamp = Light.point(listOf(0.0f, 0.0f, 0.0f), listOf(1.0f, 1.0f, 1.0f))